### PR TITLE
GEOT-5148: Switching to NetCDF 4.6.2 to get latest fixes-bkp to 13.x

### PR DESCRIPTION
**Pull request by dromagnoli**

### CODE REVIEW

**Comment by jodygarnett** [comment]
@dromagnoli can I merge this in prior to the beta?

File: modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/NetCDFImageReader.java [hunk 1]
```diff
@@ -997,6 +997,12 @@ String suggestSchemaFromCoordinateSystem(Coverage coverage, CoordinateSystem cs)
         for (CoordinateAxis axis:cs.getCoordinateAxes()) {
             // get from coordinate vars
             final CoordinateVariable<?> cv = coordinatesVariables.get(axis.getFullName()); 
+            if (cv == null) { 
+                if (LOGGER.isLoggable(Level.FINE)) {
+                    LOGGER.fine("Unable to find a coordinate variable for " + axis.getFullName());
+                }
+                continue;
+            }
```
Comment (jodygarnett) [inline modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/NetCDFImageReader.java]: Update file header

**Comment by dromagnoli** [comment]
Yep. Also waiting for more feedbacks on 4.6.2 from master before backporting.

**Comment by aaime** [comment]
Hi Daniele, we are well beyond the one month mark. Do you want to merge it?

**Pull request merged**